docs/UserGuide/Cluster/Cluster-Setup-Example.md

```diff
@@ -65,4 +128,90 @@ nohup ./cluster/target/iotdb-cluster-0.13.0-SNAPSHOT/sbin/start-node.sh ./cluste
 nohup ./cluster/target/iotdb-cluster-0.13.0-SNAPSHOT/sbin/start-node.sh ./cluster/target/test-classes/node3conf/ >/dev/null 2>&1 &
 ```
 ### Use the official website release version:
-You can refer to the above example, but starting multiple instances on a single machine requires handling port and file directory conflicts.
\ No newline at end of file
+
+Download the release version:
+```
+curl -O https://downloads.apache.org/iotdb/0.12.1/apache-iotdb-0.12.1-cluster-bin.zip
+```
+
+Unzip the package:
+
+```
+unzip apache-iotdb-0.12.1-cluster-bin.zip
+```
+
+Enter IoTDB cluster root directory:
+
+```
+cd apache-iotdb-0.12.1-cluster-bin
+```
+
+Set default\_replica\_num = 1:
+
+```
+sed -i -e 's/^default_replica_num=3$/default_replica_num=1/g' conf/iotdb-cluster.properties
+```
+
+Create conf\_dir for node2 and node3:
+
+```
+cp -r conf node2_conf
+cp -r conf node3_conf
+```
+
+Handle port and file directory conflicts:
+
+```
+sed -i -e 's/^internal_meta_port=9003$/internal_meta_port=9005/g' -e 's/^internal_data_port=40010$/internal_data_port=40012/g' node2_conf/iotdb-cluster.properties
```

Comment:
this can not work on MacOS, PTAL

Reply:
This script can run at ubuntu20.04. Command "sed" has some conflicts between unix and MacOS.